source/errors/generic.html.md.erb

```diff
@@ -0,0 +1,12 @@
+---
+title: Sorry, there is a problem with the service
+hide_from_sitemap: true
+hide_in_navigation: true
+index: false
```

Comment:
Again, would it make sense to use layout: core here so that the error pages don't get the navigation?

Reply:
👍